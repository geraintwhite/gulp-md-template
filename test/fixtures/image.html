<!doctype html>

<html>
  <head>{{image.md}}</head>
  <body>{{body.md}}</body>
</html>
