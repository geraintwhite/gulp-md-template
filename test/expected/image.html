<!doctype html>

<html>
  <head><div><img src="images/pebble.png" alt="Screenshot of Pebble Info"></div></head>
  <body><h1 id="heading1">Heading1</h1>
<p>A nice list of stuff:</p>
<ul>
<li>item 1</li>
<li>item 2</li>
<li>item 3</li>
</ul>
</body>
</html>
